docs/faqs.md

```diff
@@ -0,0 +1,132 @@
+
+  Answer: The following options are available for assistance:
+
+* Cumulus: File a bug
+* AWS: You can create a case in the [AWS Support Center](https://console.aws.amazon.com/support/home), accessible via your AWS Console when logged in via CloudTamer.
+
+</details>
+
+### Integrators & Developers
+
+<details>
+  <summary>What is a Cumulus integrator?</summary>
+
+  Answer: Those who are working within Cumulus and AWS for deployments and to manage workflows. They may perform the following functions:
+
+* Configure and deploy Cumulus to the AWS environment
+* Configure Cumulus workflows
+* Write custom workflow tasks
+
+</details>
+
+<details>
+  <summary>Can I connect my Cumulus instance to ESDIS Metrics?</summary>
+
+  Answer: Yes, you can integrate Cloud Metrics. View details [here](../features/distribution-metrics#esdis-metrics-in-ngap) on the options available and how to connect your instance to metrics.
+</details>
+
+<details>
+  <summary>What are the steps if I run into an issue during deployment?</summary>
+
+  Answer: If you encounter an issue with your deployment go to the [Troubleshooting Deployment](../troubleshooting/troubleshooting-deployment) guide.
+</details>
+
+<details>
+  <summary>What is a Cumulus workflow?</summary>
+
+  Answer: A workflow is a provider-configured set of steps that describe the process to ingest data. Workflows are defined using [AWS Step Functions](https://docs.aws.amazon.com/step-functions/index.html). For more details, we suggest visiting [here](../workflows/workflows-readme).
+</details>
+
+<details>
+  <summary>How do I set up a Cumulus workflow?</summary>
+
+  Answer: You will need to create a provider, have an associated collection (add a new one), and generate a new rule first. Then you can set up a Cumulus workflow by following these steps [here](../workflows/developing-a-cumulus-workflow).
+</details>
+
+<details>
+  <summary>What are the common use cases that a Cumulus integrator encounters?</summary>
+
+  Answer: The following are some examples of possible use cases you may see:
+
+* [Creating Cumulus Data Management Types](../integrator-guide/create-cumulus-data-mgmt-types)
+* [Workflow: Add New Lambda](../integrator-guide/workflow-add-new-lambda)
+* [Workflow: Troubleshoot Failed Step(s)](../integrator-guide/workflow-ts-failed-step)
+
+</details>
+
+### Operators
+
+<details>
+  <summary>What is a Cumulus operator?</summary>
+
+  Answer: Those that ingests, archives, and troubleshoots datasets (called collections in Cumulus). Your daily activities might include but not limited to the following:
+
+* Ingesting datasets
+* Maintaining historical data ingest
+* Starting and stopping data handlers
+* Managing collections
+* Managing provider definitions
+* Creating, enabling, and disabling rules
+* Investigating errors for granules and deleting or re-ingesting granules
+* Investigating errors in executions and isolating failed workflow step(s)
+
+</details>
+
+<details>
```

Comment:
This question can be deleted. This doc I think is deleted from the current version of the docs.

Reply:
Sorry, @laurenfrederick. This question didn't show up for some reason. Which one was it?

Reply:
`How do I set up an AWS CloudFront endpoint to serve the Cumulus Dashboard?`

Reply:
Thank you.